docs/core/tools/global-json.md

```diff
@@ -1,32 +1,104 @@
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+The *global.json* file allows selection of the .NET Core SDK version being used through the `sdk` property.
+
+.NET Core SDK looks for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
 
 ## sdk
+
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK.
 
 ### version
+
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
+
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
 
-For example:
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300-preview1-008174"
   }
 }
 ```
+
+You can easily create this file by executing the [dotnet new](dotnet-new.md) command as follows:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300-preview1-008174
+```
+
+## Matching rules
+
+With .NET Core SDK 1.x, if you specified a version and no exact match was found, the latest installed SDK version was used. Latest SDK version can be either release or pre-release - the highest version number wins.
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
+
+- If no *global.json* file is found or *global.json* doesn't specify a SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If *global.json* does specify a SDK version:
+  - If the specified SDK version is found on the machine, that exact version is used.
+  - If the specified SDK version cannot be found on the machine, the latest installed SDK **patch-version** of the specified SDK version is used. Latest installed SDK **patch-version** can be either release or pre-release - the highest version number wins.
+  - If the specified SDK version and an appropriate SDK **patch-version** cannot be found, an error is thrown.
+
+The SDK version is now composed of the following parts:
+`[major][minor][xyz]-[additional optional information]`
+
+The **patch-version** is defined by the third position (`z`) in the last portion of the number (`xyz`) for SDK versions earlier than 2.1.100 and the last two digits (`yz`) starting with 2.1.100. For example, if you specify `2.1.300` as the SDK version, it can try to find up to `2.1.399` but `2.1.400` is not considered a patch version for `2.1.300`.
+
+## Supported .NET Core SDK versions
+
+The following .NET Core SDK version numbers are the officially supported .NET Core SDK versions:
+
+- 2.1.300-rc1-008673
```

Comment:
This is already out of date, missing 2.1.300 RTM (and others). This is going to be a pain to get updated.

Reply:
@mairaw I don't understand your comment. What's the value of this list here?

Reply:
@KathleenDollard a customer was asking at #5250 but I didn't like it having it here so I will be removing when I finally have a chance to finish this PR. Instead, I'll add instructions about what to do when you get the preview message.